config/etc/nginx/conf.d

```diff
@@ -0,0 +1,60 @@
+#Template from https://docs.docker.com/registry/recipes/nginx/
+
+events {
+    worker_connections  1024;
+}
+
+http {
+  ## Set a variable to help us decide if we need to add the
+  ## 'Docker-Distribution-Api-Version' header.
+  ## The registry always sets this header.
+  ## In the case of nginx performing auth, the header is unset
+  ## since nginx is auth-ing before proxying.
+  map $upstream_http_docker_distribution_api_version $docker_distribution_api_version {
+    '' 'registry/2.0';
+  }
+
+  server {
+    listen 443 ssl;
+    server_name ${DOMAIN};
+
+    # SSL
+    ssl_certificate /etc/nginx/conf.d/domain.crt;
+    ssl_certificate_key /etc/nginx/conf.d/domain.key;
```

Comment:
@manekenpix where will certbot put these?

Reply:
We're setting `/etc/letsencrypt` below, but I don't know where under there it puts things.

Reply:
This should probably be like what we do for telescope's front-end, [here](https://github.com/Seneca-CDOT/telescope/blob/7ffaed87001df626cbaad05c652935b79025ad6d/config/nginx.conf.template#L138).
I think we'll have to change the docker file to match this.